internal/journal/voljournal.go

```diff
@@ -608,12 +613,14 @@ func (conn *Connection) GetImageAttributes(ctx context.Context, pool, objectUUID
 	values, err := getOMapValues(
 		ctx, conn, pool, cj.namespace, cj.cephUUIDDirectoryPrefix+objectUUID,
 		cj.commonPrefix, fetchKeys)
-	switch err.(type) {
-	case nil:
-	case util.ErrPoolNotFound, util.ErrKeyNotFound:
-		klog.Warningf(util.Log(ctx, "unable to read omap keys: pool or key missing: %v"), err)
-	default:
-		return nil, err
+	if err != nil {
+		var eknf util.ErrKeyNotFound
+		var epnf util.ErrPoolNotFound
+		if errors.As(err, &eknf) || errors.As(err, &epnf) {
```

Comment:
can we remove else block if we change this to
```go
if !errors.As(err, &eknf) && !errors.As(err, &epnf) {
return nil error
}
 klog.Warningf(util.Log(ctx, "unable to read omap keys: pool or key missing: %v"), err)
```

Reply:
sure